examples/logging/README.md

```diff
@@ -0,0 +1,340 @@
+# DeepSparse Logging
+
+DeepSparse Logging provides operational teams with access to the telemetry needed to monitor a deployment. 
+
+For users seeking to put ML into production, these data are the raw materials that underpin the monitoring processes needed to create a system with consistently fresh and accurate predictions.
```

Comment:
Nit: do not like the "raw materials" wording here. Feels odd. Maybe the "essential source of truth" or smth?